### ISSUE TITLE
Itemset showed is wrong

### ISSUE THREAD

**Issue by Helias** [Sarjuuk/aowow#195]
Currently, the items listed in as itemset do not correspond with the correct one.

AoWoW: https://wowgaming.altervista.org/aowow/?item=51413
Expected: https://www.wowhead.com/item=51413/wrathful-gladiators-dreadplate-chestpiece

The problem is in item.class.php line 1039.
That query is wrong, you should select all items with different slotBak and belong to the same item ID "range".

To select the correct list of items for 51413 you can try this query:
```SQL
SELECT * FROM items WHERE itemset=768 GROUP BY slotBak;
```

**Comment by Helias** [comment]
A good solution could be the following.

Use this query:
```SQL
SELECT * FROM items WHERE itemset = ?a ORDER BY slotBak, id;
```

Detect the order of the selected item, and select the other item of the itemset based on the order.

Example: **itemID = 51413, itemset = 768**
Run the following query:
```SQL
SELECT * FROM items WHERE itemset = 768 ORDER BY slotBak, id;
```

The itemID of the example is in the 7 row compared with the other slotBak, the other items which belong to the same itemset of this item are in the 7 position.

(before 40781 there are 2 items with slotBak = 5)
![image](https://user-images.githubusercontent.com/519778/77484020-ce2d2100-6e29-11ea-93fb-0417de6a6efa.png)

The same logic works with the others itemset.

**Comment by Sarjuuk** [comment]
Jebus, that took me longer to decipher than it should have. 

The Itemset listed in the Tooltip is for the wrong item level group. 
Is that correct?

Also, there is no query anywhere near the file+line you mentioned: https://github.com/Sarjuuk/aowow/blob/master/includes/types/item.class.php#1039

**Comment by Helias** [comment]
Sorry, I was using as reference of the project this https://github.com/azerothcore/www-aowow/

The line where you can find the query is this https://github.com/Sarjuuk/aowow/blob/master/includes/types/item.class.php#L986

**Comment by Sarjuuk** [comment]
alright.

fixed in 3b749025de319a4583af23d64c59bb041d79fc1a
